File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ManualTestSlide.java
package org.firstinspires.ftc.teamcode;

import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
import com.qualcomm.robotcore.eventloop.opmode.TeleOp;

@TeleOp(name = "Test slide manually")
public class ManualTestSlide extends LinearOpMode {
	@Override
	public void runOpMode() {
		Slide slide = new Slide(hardwareMap, telemetry, false);
		Claw claw = new Claw(hardwareMap, telemetry);
		GamepadEx p1 = new GamepadEx();

		waitForStart();

		while (opModeIsActive()) {
			p1.tick(gamepad1);
			slide.control(p1);
			claw.control(p1);
		}
	}
}

File: TeamCode/src/main/kotlin/org/firstinspires/ftc/teamcode/ManualTestSlide.kt
package org.firstinspires.ftc.teamcode

import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode
import com.qualcomm.robotcore.eventloop.opmode.TeleOp
import com.qualcomm.robotcore.hardware.DcMotorEx

@TeleOp(name = "Manual slide test", group = "Manual test")
class ManualTestSlide : LinearOpMode() {
	override fun runOpMode() {
		val slide = hardwareMap.get(DcMotorEx::class.java, "slide")
		slide.configure()
		val positionItem = telemetry.addData("Position", "")

		waitForStart()

		while (opModeIsActive()) {
			slide.power = (gamepad1.right_trigger - gamepad1.left_trigger).toDouble()
			positionItem.setValue(slide.currentPosition)
			telemetry.update()
		}
	}
}